libs/ledger-live-common/src/families/evm/__tests__/integration/bridge.integration.test.ts

```diff
@@ -1,21 +1,20 @@
-import "../../__tests__/test-helpers/setup";
+import "../../../../__tests__/test-helpers/setup";
 
 import { BigNumber } from "bignumber.js";
+import { FeeNotLoaded, NotEnoughBalance } from "@ledgerhq/errors";
 import type { AccountRaw, DatasetTest } from "@ledgerhq/types-live";
-import { FeeTooHigh } from "@ledgerhq/errors";
-import { testBridge } from "../../__tests__/test-helpers/bridge";
-import type { Transaction } from "./types";
-import { fromTransactionRaw } from "./transaction";
-import ethereumScanAccounts1 from "./datasets/ethereum.scanAccounts.1";
-import { ethereum1 } from "./datasets/ethereum1";
+import { testBridge } from "../../../../__tests__/test-helpers/bridge";
+import { fromTransactionRaw } from "../../transaction";
+import { ethereum1 } from "../../datasets/ethereum1";
+import type { Transaction } from "../../types";
 
 const dataset: DatasetTest<Transaction> = {
   implementations: ["mock", "js"],
   currencies: {
-    ethereum: {
-      scanAccounts: [ethereumScanAccounts1],
+    ethereum_as_evm_test_only: {
       accounts: [
         {
+          FIXME_tests: ["balance is sum of ops"],
```

Comment:
By curiosity, is it failing ? If yes, do we have an idea why ?

Reply:
Yes it is, no idea why but that's a very difficult test to have correct anyway. I'll have a look later to find out what is missing exactly (could be an unsupported operation type, could be some fee not accounted correctly), but that test is deactivated for Ethereum/Ripple/Celo/Cosmos/Filecoin/Near & Stellar, so I guess either it doesn't make sense anymore, or we need to rework it globally.